### ISSUE TITLE
Any domain resource once published then removed from any level of accessibility should have a gravestone message

### ISSUE THREAD

**Issue by cmh2166** [sul-dlss/taco-truck#78]
REQUIREMENT: Any domain resource once published then removed from any level of accessibility should have a gravestone message.
* Idea behind this requirement: This is best practice for repository management, so that anyone returning to a domain resource on Access & Discovery Interface side is given an indicator that the resource was deleted, so it doesn’t just look like a 404 / error.
* Need to clarify what leads to this and access-only MAP / data shapes (i.e. object is still in management but needs Access to only retrieve metadata saying ‘this object is no longer available’).
* QUESTION: Graveyard / Tombstone / Gravestone / Citation only in this context / Tombstone message: We need to define these.
* REQUEST: Make decommissioned resource rollbacks feasible. 
* Reversible by reconstruction of a preserved copy to penultimate version within Management. Relates to preservation and versioning questions (below) - i.e. what is retrieved from where for rollback and what counts as a new version.
